docs/guides/elasticstack-cloud-with-monitoring.md

```diff
@@ -43,118 +45,24 @@ attribute to target the intended deployment. Aliased providers can be configured
 For more information consult the [documentation](https://developer.hashicorp.com/terraform/language/providers/configuration#alias-multiple-provider-configurations)
 
 ```terraform
-# Creating a deployment on Elastic Cloud GCP region,
-# with elasticsearch and kibana components.
-
```

Comment:
This example no longer makes sense since the actual detail has been moved to other files which aren't included in the generated docs.